website/docs/index.html.markdown

```diff
@@ -219,20 +247,27 @@ The following arguments are used to configure the VMware Cloud Director Provider
 * `password` - (Required) This is the password for Cloud Director API operations. Can
   also be specified with the `VCD_PASSWORD` environment variable.
 
-* `auth_type` - (Optional) `integrated`, `token` or `saml_adfs`. Default is `integrated`.
+* `auth_type` - (Optional) `integrated`, `token`, `api_token`, or `saml_adfs`. Default is `integrated`.
   * `integrated` - VCD local users and LDAP users (provided LDAP is configured for Organization).
   * `saml_adfs` allows to use SAML login flow with Active Directory Federation
   Services (ADFS) using "/adfs/services/trust/13/usernamemixed" endpoint. Please note that
   credentials for ADFS should be formatted as `user@contoso.com` or `contoso.com\user`. Can also be
   set with `VCD_AUTH_TYPE` environment variable.
   * `token` allows to specify token in [`token`](#token) field.
+  * `api_token` allows to specify an API token.
   
 * `token` - (Optional; *v2.6+*) This is the token that can be used instead of username
    and password (in combination with field `auth_type=token`). When this is set, username and
    password will be ignored, but should be left in configuration either empty or with any custom
    values. A token can be specified with the `VCD_TOKEN` environment variable.
    Both a (deprecated) authorization token or a bearer token (*v3.1+*) can be used in this field.
 
+* `api_token` - (Optional; *v3.5+*) This is the API token that a System or organization administrator can create and 
```

Comment:
I think we need to add additional explanation how `token` differs from `api_token`. We have two tokens now :) Not sure in which field it fits better though.

Reply:
The text has some explanation and the example provides even more information.
We can't supply all the documentation here, but I added some wording to differentiate the two items.

Reply:
Thanks, it's clearer now!